server/server.js

```diff
@@ -17,7 +18,11 @@ app.get('/home', (req, res) => {
     res.send(`Hello`);
 })
 
-app.post('/users/signup', async (req, res) => {
+app.post('/users/signup', check("email", "This is not a valid email").isEmail(), async (req, res) => {
```

Comment:
Looks good!

Let's move users route to a single file using `express.Router()` and communicate those with `app.use` within `server.js`